src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -190,6 +190,11 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             {
                 defaultSettings.ReloadEnvironmentVariables(newTerminalArgs.ReloadEnvironmentVariables().Value());
             }
+
+            if (newTerminalArgs.KeepOpen())
+            {
+                // defaultSettings.CloseOnExit = CloseOnExitMode::Never;

```

Comment:
This line is commented as it brakes the build process. VS says that `defaultSettings` has no `CloseOnExit` property and I don´t know why. How can I manipulate this setting if the comandline argument is present?